src/NewSessionView.elm

```diff
@@ -54,6 +54,28 @@ view context model =
                 Just val ->
                     True
 
+        allColumnsDropdownOption =
+            option [ value ("ALL COLUMNS"), selected (context.session.sessionColumn == AllColumns) ] [ text "ALL COLUMNS" ]
+
+        columnOptions =
+            case context.session.sessionColumn of
+                ColumnId columnIdInt ->
+                    (List.map (\c -> option [ value (toString c.id), selected (columnIdInt == c.id) ] [ text c.name ]) model.columns)
+
+                _ ->
+                    (List.map (\c -> option [ value (toString c.id) ] [ text c.name ]) model.columns)
+
+        selectedAllColumnsValue =
```

Comment:
can this be one line: `context.session.sessionColumn == AllColumns`?
And if that's the case, is this variable needed?